webapp/src/main/webapp/vue-apps/CallButtons/components/CallButtons.vue

```diff
@@ -323,8 +323,6 @@ export default {
       width: 100%;
     }
     cursor: pointer !important;
-    position: relative;
```

Comment:
IMO position relative has no effect on z-index attribute, so no need to delete this line @boubaker WDYT ?

Reply:
Yep, this is a cleanup of useless CSS properties